consensus/misc/kip71.go

```diff
@@ -24,14 +24,15 @@ func VerifyKIP71Header(config *params.ChainConfig, parentHeader, header *types.H
 }
 
 func NextBlockBaseFee(parentHeader *types.Header, config *params.ChainConfig) *big.Int {
+	// governance parameters
+	lowerBoundBaseFee := new(big.Int).SetUint64(config.Governance.KIP71.LowerBoundBaseFee)
+	upperBoundBaseFee := new(big.Int).SetUint64(config.Governance.KIP71.UpperBoundBaseFee)
+
 	// If the parent is the kip71 disabled block or genesis, then return default base fee (250ston)
 	if !config.IsKIP71ForkEnabled(parentHeader.Number) || parentHeader.Number.Cmp(new(big.Int).SetUint64(0)) == 0 {
-		return new(big.Int).SetUint64(config.UnitPrice)
+		return lowerBoundBaseFee
```

Comment:
Can we say that lowerBoundBaseFee == 250 ston?

Reply:
Ahh, the comment should be edited by 25ston, Thanks!

Reply:
It's a little confusing to me what the 'default base fee' refers to.

- If you intention is to return lowerBoundBaseFee (config.Governance.KIP71.UpperBoundBaseFee), then the comment can be
  ```go
  // If the parent is the kip71 disabled block or genesis, then return the LowerBoundBaseFee (default 25 ston)
  ```
- If your intention is to return default base fee (params.DefaultLowerBoundBaseFee = 25ston) then the code should be
  ```go
  return params.DefaultLowerBoundBaseFee
  ```

Reply:
Ok, I intended the former. fixed it.

Reply:
@kjeom Hmm? really? anyway current gas price is 250 ston and before KIP-17 is enabled, we should return 250 ston, shouldn't we?

Reply:
@kjeom Then, why we need to check `config.IsKIP71ForkEnabled(parentHeader.Number)` if you're sure that this will not be called if KIP-71 is not enabled?

Reply:
Okay. got it. Thanks.